requirements.txt

```diff
@@ -5,6 +5,7 @@ jinja2
 murl
 pyapi-gitlab
 requests
+requests-mock
```

Comment:
This should go in `requirements-dev.txt` since it is a testing requirement

Reply:
nice catch